lib/bloc/account/credentials_manager.dart

```diff
@@ -50,4 +52,10 @@ class CredentialsManager {
       throw e.toString();
     }
   }
+
+  Future<StaticBackupResponse> exportStaticChannelBackup() async {
+    final breezLib = ServiceInjector().breezSDK;
+    Directory workingDir = await getApplicationDocumentsDirectory();
```

Comment:
Better to take the workingDir from the config rather than recalculate it here.

Reply:
@roeierez  It seems to me that getting the `workingDir` from the config is doing the same call as here pluss some more. 
https://github.com/breez/c-breez/blob/f10bb948ea2c115b4362bc12d93f915f58c1c789/lib/config.dart#L125-L129.
Is it not better to use the `getApplicationDocumentsDirectory()` or am I missing something ?

Reply:
It is better to keep a single source of truth for the wirkingDir

Reply:
Good point, added.